numerapi/numerapi.py

```diff
@@ -1103,3 +1103,19 @@ def daily_submissions_performances(self, username: str) -> List[Dict]:
         performances = [p for p in performances
                         if any([p['correlation'], p['fnc'], p['mmc']])]
         return performances
+
+    def get_dataset_list(self):
+        query = """
+            query { listDatasets }
+        """
+        return self.raw_query(query)['data']['listDatasets']
+
+    def download_dataset(self, filename: str, dest_path: str):
```

Comment:
again, missing doc string, missing type hints

Reply:
Isn't this doing the same as  `Numerapi.download_latest_data`?  there is also `Numerapi.get_dataset_url`, which should be re-used

Reply:
this gets a single file instead of the zip

Reply:
or, if the other two methods I mentioned are deprecated. Let's mark them as such